### PR TITLE
Landing page self-hosted vs Cloud clarifications

### PR DESCRIPTION
**Pull request by cwarnermm**
Updated the Administration landing page to clarify self-hosted vs Cloud-specific features, functionality, and configuration options.

### CODE REVIEW

File: source/guides/administration.rst [hunk 1]
```diff
@@ -3,15 +3,15 @@ Mattermost administration guide
...
 * `Other resources <https://docs.mattermost.com/guides/administration.html#other-resources>`__ - Additional migration, conversion, and troubleshooting resources.
 
-If you’re looking for resources to help you install, deploy, and scale your self-hosted Mattermost server, refer to the :doc:`Mattermost Deployment Guide </guides/deployment>`.
+If you’re looking for resources to help you install, deploy, and scale your self-hosted Mattermost server, see the :doc:`Mattermost Deployment Guide </guides/deployment>`.
 
-The basics
-----------
+Get started
+------------
 .. toctree::
```
Comment (jasonblais) [inline source/guides/administration.rst:15]: Does this TOC still apply? I noticed it doesn't match the descriptions from line 39 onward

File: source/guides/administration.rst [hunk 2]
```diff
@@ -34,197 +34,131 @@ The basics
...
+* **Configure your workspace** - Mattermost offers extensive configuration options for both `self-hosted Mattermost servers and Mattermost Cloud workspaces <https://docs.mattermost.com/configure/configuration-settings.html>`__.
+* **Provisioning workflows** - Learn how to :doc:`provisioning </onboard/user-provisioning-workflows>` and de-provision user accounts.
+* **User authentication** - All versions of Mattermost provide basic authentication, :doc:`multi-factor authentication </onboard/multi-factor-authentication>`, and :doc:`GitLab SSO </onboard/sso-gitlab>` out of the box. Professional and Enterprise versions of Mattermost also include :doc:`Active Directory/LDAP </onboard/ad-ldap>` and SSO for :doc:`OpenID </onboard/sso-openidconnect>`, :doc:`Google </onboard/sso-google>`, and :doc:`Office365 </onboard/sso-office>`.
```
Comment (jasonblais) [inline source/guides/administration.rst:42]: GitLab is only Professional or Enterprise in Cloud

File: source/guides/administration.rst [hunk 3]
```diff
@@ -34,197 +34,131 @@ The basics
...
     :maxdepth: 1
     :hidden:
 
     AD/LDAP groups </onboard/ad-ldap-groups-synchronization>
+    Use AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     Guest accounts </onboard/guest-accounts>
-    Using AD/LDAP synchronized groups to manage team or private channel membership </onboard/managing-team-channel-membership-using-ad-ldap-sync-groups>
     SAML Single Sign-On </onboard/sso-saml>
     SAML Single-Sign-On: technical documentation </onboard/sso-saml-technical>
     SSL client certificate setup </onboard/ssl-client-certificate>
```
Comment (jasonblais) [inline source/guides/administration.rst:60]: Are SSL client certificate and certificate-based auth available in Cloud? I'm not sure

File: source/guides/administration.rst [hunk 4]
```diff
@@ -34,197 +34,131 @@ The basics
...
 .. toctree::
     :maxdepth: 1
     :hidden:
 
-    Managing team and channel members </manage/team-channel-members>
+    Shared channels </onboard/shared-channels>
     Statistics </manage/statistics>
     In-product notices </manage/in-product-notices>
     User satisfaction surveys </manage/user-satisfaction-surveys>
     Health check </manage/health-checks>
```
Comment (jasonblais) [inline source/guides/administration.rst:82]: not sure if the health check works for Cloud -- looking at the docs it likely will, but would be good to confirm

**Review by jasonblais — COMMENTED** [review]
@cwarnermm this is awesome! Just a few smaller notes, this will be a great update :+1:

File: source/guides/administration.rst (hunk 3 above)
Comment (cwarnermm) [reply source/guides/administration.rst:60]: Confirmed with Maria that these two self-hosted features are not supported via Cloud instances.

File: source/guides/administration.rst (hunk 4 above)
Comment (cwarnermm) [reply source/guides/administration.rst:82]: Confirmed with Maria that health check is supported via Cloud instances. The endpoint is available for cloud instances and administrator could configure their pinging checks for their cloud servers to make sure it's always up and running.

File: source/guides/administration.rst (hunk 1 above)
Comment (cwarnermm) [reply source/guides/administration.rst:15]: Thanks, @jasonblais - it does still apply, and I agree that the LHS and the landing page need to be more parallel. It's important that users can reach individual pages of interest through both the landing page and the LHS on demand. Good catch!

**Comment by cwarnermm** [comment]
Great feedback, @jasonblais! I've incorporated all of your points in my latest commit.

**Review by jasonblais — APPROVED** [review]
Thanks Carrie!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**